Makefile

```diff
@@ -23,7 +23,8 @@ e2e-helm-deploy:
 	mkdir -p .staging/helm
 	curl https://get.helm.sh/helm-v${HELM_VERSION}-linux-amd64.tar.gz > .staging/helm/helmbin.tar.gz
 	cd .staging/helm && tar -xvf helmbin.tar.gz
-	./.staging/helm/linux-amd64/helm install osm ./charts/osm --namespace arc-osm-system
+	./.staging/helm/linux-amd64/helm dependency update ./charts/osm-arc
+	./.staging/helm/linux-amd64/helm install osm ./charts/osm-arc --namespace arc-osm-system
 
 test-e2e:
```

Comment:
We should add a new Makefile target similar to this [example](https://github.com/Azure/secrets-store-csi-driver-provider-azure/blob/master/Makefile#L130-L139). We can run this target before cutting a release of osm-arc to ensure the osm chart has been updated and included as part of the new release of osm-arc chart.

Reply:
created https://github.com/Azure/osm-azure/issues/21